doc/authentication-methods/client-certificate.md

```diff
@@ -46,11 +46,33 @@ Below you may find a list of backends that are safe to use with `cyrsasl_externa
 * `rdbms`
 * `riak`
 
+### Self-signed certificates
+
+By default MongooseIM doesn't accept self-signed certs for the SASL-EXTERNAL authentication.
+They may be handy in development so we made it possible to tell MongooseIM to accept them.
```

Comment:
They may be handy in development, so we made telling MongooseIM to accept them possible.
They may be handy in development, so we enabled telling MongooseIM to accept them.
For development purposes, it is possible to tell MongooseIM to accept them.
We made it possible to tell MongooseIM to accept them because it might come handy for development purposes.

?